.travis.yml

```diff
@@ -1,7 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - '6'
-  - '4'
-  - '0.12'
-  - '0.10'
+  - 4
+  - 6
+  - 7
```

Comment:
curious. none of my other projects have these quoted and travis works as expected. please do share why